store/user.ts

```diff
@@ -1,55 +1,30 @@
 import { ActionTree, MutationTree } from 'vuex';
 
 export interface UserState {
-  id: string;
   accessToken: string;
```

Comment:
Not really, because the way Nuxt Auth checks for login is by using that $auth store (fetchUser in the scheme stores the UserDto there, which then means we are logged in).

The ideal would be to get rid of the user store completely and get the access token (which is the only thing we use it for, now) as part of the UserDto.

Reply:
Ah ok. My only thought was we might get type safety if it was stored in the user store instead of $auth